lib/chef-acceptance/cli.rb

```diff
@@ -13,17 +13,23 @@ class Cli < Thor
     # Create core acceptance commands
     #
     AcceptanceCookbook::CORE_ACCEPTANCE_RECIPES.each do |command|
-      desc "#{command} TEST_SUITE_REGEX", "Run #{command}"
+      desc "#{command} TEST_SUITE_REGEX [--log-dir=/log/directory]", "Run #{command}"
+      option :log_dir,
```

Comment:
You can create shared options like this:
https://github.com/chef/chef-acceptance/pull/35/files#diff-de24f25c71b721c86ae7358cc09d5200R16
https://github.com/chef/chef-acceptance/pull/35/files#diff-de24f25c71b721c86ae7358cc09d5200R33
This is a slightly more DRY approach to the duplicate options.